phys2bids/physio_obj.py

```diff
@@ -228,6 +228,21 @@ class BlueprintInput():
     def __init__(self, timeseries, freq, ch_name, units, trigger_idx,
                  num_timepoints_found=None, thr=None, time_offset=0):
         """Initialise BlueprintInput (see class docstring)."""
+        trigger_names_list = ["trig", "trigger", "TRIGGER", "Trigger"]
+        if trigger_idx == 0:
+            name_not_found = True
+            LGR.warning('User did not input chtrig. Trying to find in auto mode by name')
+            for trig_name in trigger_names_list:
+                if trig_name in ch_name:
+                    trigger_idx = ch_name.index(trig_name)
```

Comment:
Do we want a complete string match or do we want to support partial matches? If the latter, then I think a regular expression would probably be easier here. If the former, then I'd switch the comparison (loop through `ch_name` and see if each name is "in" `trigger_names_list`, probably with a list comprehension). We also want to raise an exception or at least a warning if multiple channel names match the criteria.

Reply:
I agree! See my comment above.